build.gradle

```diff
@@ -0,0 +1,114 @@
+buildscript {
+  ext.versions = [
```

Comment:
For comparison with the Kotlin DSL

https://github.com/yschimke/okurl/blob/master/buildSrc/src/main/kotlin/Deps.kt